relay-dynamic-config/src/defaults.rs

```diff
@@ -416,13 +416,13 @@ fn span_metrics() -> impl IntoIterator<Item = MetricSpec> {
                 Tag::with_key("transaction.op")
                     .from_field("span.sentry_tags.transaction.op")
                     .always(),
+                Tag::with_key("span.group")
+                    .from_field("span.sentry_tags.group")
+                    .always(),
```

Comment:
Won't this extract the span group for every span? The exclusive time metric only extracts it for `know_modules_condition`

Reply:
+1

Reply:
I was thinking the group is calculated only for known modules, otherwise left `None`, and we wouldn't tag a `None` value.

In the end, we may calculate the group for more than what we support and then have another layer of conditions so, using the `known_modules_condition` is safer indeed.